form/src/components/Result.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from "react";
+
+const Result = ({ question }) => {
+  const [query, setQuery] = useState("");
+  const [answer, setAnswer] = useState("");
+
+  let arr = [];
```

Comment:
Don't prefer imperative programming (for loops, ara.push, arr.pop etc.) Plus `arr` isn't used at all. It's unused variable